### DISCUSSION TITLE
🌹 ROSE Bloom Grants • Build an NFT app on the Oasis Ethereum ParaTime

### DISCUSSION THREAD

**Discussion by jcueto** [oasisprotocol/community-forum#23] [Grants Program]
![image](https://user-images.githubusercontent.com/5066732/106105783-6276ad00-60f9-11eb-8a08-6b07d598e61e.png)
-----
### Apply here: https://oasisfoundation.typeform.com/to/HtYql2aN
-----

_Note: This discussion thread serves as a hub for grant applications related to this project category. You'll see project proposals that have been submitted and reviewed by the Oasis Foundation team. You'll see proposals that have been marked as "ACCEPTED" and "TURNED DOWN," which will give you a better sense of what we're looking for in a grant application. Community members are encouraged to upvote projects, provide feedback, ask questions, and offer suggestions on each project shared on this thread._ 

-----

The ROSE Bloom Grants Program is an entirely new grants program designed to accelerate the growth of the Oasis Network and build valuable new ecosystem tools. Apply for a grant today to bring your project to life!

For this specific grant category, we’re looking for developers who want to build new NFT (nonfungible token) applications on the Oasis Ethereum ParaTime: https://www.oasiseth.org/#developers  

To qualify for a grant, an application has to focus on NFTs. You can port an existing NFT-related Solidity smart contract from Ethereum or create a brand new NFT-related Solidity smart contract to be launched on the Oasis Network. You can build an NFT minting app, an NFT-focused web app or mobile app, such as a gallery or shop to showcase and sell NFT artwork, a decentralized NFT auction system, or a game with NFT collectibles or NFT assets, such as virtual real estate. Your application could be primarily UI-focused, such as an NFT art gallery, but it *must* integrate with the Oasis Ethereum ParaTime in some capacity (such as directly connecting to NFTs created on the Oasis Ethereum ParaTime). For some additional reading on the types of use cases made possible by NFTs, we recommend checking out [this post](https://101blockchains.com/nft-use-cases/). For more ideas on what you can build with the Oasis Ethereum ParaTime, check out the “HELPFUL DOCUMENTATION” section of this post.

Second State’s Oasis Ethereum ParaTime is an Ethereum backwards compatible ParaTime with support for the entire Solidity toolkit. The ParaTime is nearly identical to Ethereum, but has better performance and lower gas fees. With the Oasis Ethereum ParaTime, you can port over Solidity smart contracts from Ethereum to the Oasis Network, and you can easily launch new DeFi apps, NFT apps, and more on the Oasis Network. If you’re already familiar with building apps with Solidity, the Oasis Ethereum ParaTime lets you dive right into the Oasis Network ecosystem. Thee Oasis Ethereum ParaTime is fully compatible with Ethereum’s ERC-20 and ERC-721 token standards, as well as everything else you need to launch your very own NFT application. 

-----

### HOW TO APPLY

Applying for a grant is easy. Just fill out the [application form](https://oasisfoundation.typeform.com/to/HtYql2aN) with the relevant information we are requesting, and we’ll notify you via email regarding the status of your application. 

The more specific and clear you are in your application, the better our team will be able to understand your project proposal.

-----

### GRANT AMOUNT

Each proposal will be eligible to receive anywhere from **$5,000 to $50,000 USD** given out in **ROSE tokens**. Please specify the grant amount you are requesting in your application. The grant amount requested should match the scope of work for each project. 

-----

### PROJECT MILESTONES

We ask that you provide specific project milestones as part of your application. Milestones should be decided based on the type of project you are developing and the scope of work for the project. In general, the types of things you can list as project milestones include specific development steps (e.g. initial deployment, debugging, obtaining first user, etc.), specific components of the application (e.g. sign-in flow, front-end user interface, database config, etc.), as well as key success metrics (e.g. app installs, number of transactions, number of users, etc.).

Depending on the size of your grant, the Oasis Foundation may disburse your grant in separate parts. Smaller grants may be disbursed in a single transaction. Larger grants may be disbursed in separate transactions at separate times, based on evidence of your continued progress building your project and meeting the milestones we agreed upon. 

Please note that we expect a commitment of at least around 3 months, including project development, launch, maintenance, etc. If you want to continue working on your project or developing extensions for it beyond this initial commitment, you can apply for additional grants in the future to fund your continued work. 

-----

### SELECTION CRITERIA

The Oasis Foundation team will assess your application based on the following scoring categories: 
- Strength of the team, based on past projects, resume, GitHub, etc. - 1 (Very Weak) to 5 points (Very Strong) 
- Written quality of project proposal, based on details described, content, etc. - 1 (Very Low Quality) to 5 points (Very High Quality)
- Tech & product quality of project, based on technical implementation details and product vision, design overview, etc. - 1 (Very Low Quality) to 5 points (Very High Quality)
- Relevance to the success of the network, based on whether this is a product that could help us grow, an app that is in-demand among our community members, etc.- 1 (Not very relevant at all) to 5 points (Very Relevant)

Community interest and excitement generated will be a factor considered in the selection process, as an input into the “Relevance to the success of the network” scoring category. We’ll post project overviews on the Oasis GitHub discussion board to allow Oasis community members to share feedback, ask questions, and offer suggestions for each project. 

-----

### APPLICATION TIMELINE 

Grant applications will be reviewed on an ongoing, rolling basis. In general, the application process will follow the timeline below: 
- Apply for a grant via our application form: https://oasisfoundation.typeform.com/to/HtYql2aN 
- Allow 1 to 2 weeks for the Oasis team to do a preliminary review of your application
- If your project idea looks promising, the Oasis team will post your project overview on the Oasis GitHub discussion board and notify you via email (applicants that are turned down at this preliminary stage will also be notified). This will allow the community to share feedback on your project proposal, ask questions, and offer you suggestions.
- Allow 1 to 2 weeks after your project gets posted on the Oasis GitHub discussion board and receives community feedback for the Oasis team to mark your project as ACCEPTED or TURNED DOWN. We will also notify you if your application is approved or turned down at this stage via email.
- If your grant application is accepted, the Oasis team will email you with next steps, including information about the KYC process.
The Oasis team will work closely with you to finalize the terms of your grant, and you can begin working on your project!

-----

### HELPFUL DOCUMENTATION

The following documentation should help you as you’re building your app:
- Oasis Ethereum ParaTime Website: https://www.oasiseth.org/#developers 

Check out the following tutorials and app demos:
- Oasis Ethereum ParaTime Tutorial - https://youtu.be/NEo7W_qgKos 

For more ideas on the types of projects you can build using the Oasis Ethereum ParaTtime, you can check out some of the projects that won The Quest for ROSE: An Oasis Network Hackathon: https://medium.com/oasis-protocol-project/announcing-the-winners-of-the-quest-for-rose-hackathon-5d160722a840 .

-----

### GET SUPPORT

- Join our Oasis Community Slack to view all of our support channels for developers: https://join.slack.com/t/oasiscommunity/shared_invite/zt-8a71w8i9-sVhe_5IG1hhb5Dgjl5xu2A 
- For logistical questions related to the ROSE Bloom Grants Program, post in the #grants-program-public channel.
- For tech questions related to building with the Oasis Ethereum ParaTime, post in the #oasiseth-paratime channel
- To learn more about the Oasis Network community of developers and supporters, join our Telegram: https://t.me/oasisprotocolcommunity 

-----

### TERMS & CONDITIONS

Please note that eligibility to get approved for a grant is subject to KYC and your country of residence's accredited investor verification guidelines, as outlined in the official program rules (https://docsend.com/view/xykrayt6vb5tn7eh). 

Developers who are U.S. citizens or residents and who are not accredited investors may not be eligible to receive ROSE Bloom Grants. The grants are not and will not be offered or sold, directly or indirectly, to any person who is a resident, organized, or located in any country or territory subject to OFAC comprehensive sanctions programs from time to time, including Cuba, Crimea region of Ukraine, Democratic People’s Republic of Korea, Iran, Syria, any person found on the OFAC Specially Designated Nationals, Blocked Persons List, any other consolidated prohibited persons list as determined by any applicable governmental authority, or any persons who are organized in, located in, or residents of New York State.

-----

**Comment by dhanushreddy291** [comment]
Is the proposal still on?
